pkg/lvm/lvm_util.go

```diff
@@ -52,13 +52,41 @@ const (
 	LVCreate = "lvcreate"
 	LVRemove = "lvremove"
 	LVExtend = "lvextend"
-	LVS      = "lvs"
+	LVList   = "lvs"
 
 	PVScan = "pvscan"
 
 	YES = "yes"
 )
 
+// LogicalVolume specifies attributes of a given lv that exists on the node.
+type LogicalVolume struct {
+
+	// Name of the lvm logical volume(name: pvc-213ca1e6-e271-4ec8-875c-c7def3a4908d)
+	Name string
+
+	// Full name of the lvm logical volume (fullName: linuxlvmvg/pvc-213ca1e6-e271-4ec8-875c-c7def3a4908d)
+	FullName string
+
+	// UUID denotes a unique identity of a lvm logical volume.
+	UUID string
```

Comment:
I think this field is not getting populated while lvm parsing.

Reply:
UUID and fullName both are not getting populated.

Reply:
Is it not required for the metrics?

Reply:
No, for now, it's not required.